src/index.html

```diff
@@ -72,11 +72,9 @@ <h3 class="card__name">
           <p class="card__price__value">69 999 грн</p>
         </div>
 
-        <a
-          href="#"
+        <a href="#"
           class="card__button"
-          data-qa="card-hover"
-        >
+          data-qa="card-hover">
```

Comment:
incorrect correction, closing braket ">" should be alined as open tag, look https://mate-academy.github.io/style-guides/htmlcss.html#good